doc-content/enterprise-only/processes/pizza-orders-task-proc.adoc

```diff
@@ -14,7 +14,6 @@ image::processes/create-task2.png[Creating an outgoing connection from the start
 .Convert into a User task
 image::processes/user_task2.png[Converting in to a user task]
 
-. In the upper-right corner of the the process designer, click the *Diagram properties* image:processes/diagram_properties.png[] icon.
```

Comment:
Same as before: not sure, why is it removed. For new users it can be useful.

Reply:
@hasys Diagram properties are already opened. If you click it again you will close it.